src/cocotb/share/include/py_gpi_logging.h

```diff
@@ -23,6 +23,8 @@ PYGPILOG_EXPORT void py_gpi_logger_initialize(PyObject* handler,
 
 PYGPILOG_EXPORT void py_gpi_logger_finalize();
 
+extern PYGPILOG_EXPORT PyObject* pEventFn;  // This is gross but I don't care
```

Comment:
This file is currently the only one both linked by libcocotb (gpi_embed.cpp) and simulator. But in the (hopefully near) future, all three of these files will be merged into one library.